src/pymmcore_widgets/hcs/_hcs_wizard.py

```diff
@@ -0,0 +1,241 @@
+        # add custom button to load
+        self.setOption(QWizard.WizardOption.HaveCustomButton2, True)
+        if load_btn := self.button(QWizard.WizardButton.CustomButton2):
+            load_btn.setText("Load")
+            load_btn.clicked.connect(self.load)
+            self.setButton(QWizard.WizardButton.CustomButton2, load_btn)
+
+        # CONNECTIONS ---------------------------
+
+        self.plate_page.widget.valueChanged.connect(self._on_plate_changed)
+        self._on_plate_changed(self.plate_page.widget.value())
+        self.calibration_page.widget.calibrationChanged.connect(
+            self._on_calibration_changed
+        )
+
+    def sizeHint(self) -> QSize:
+        return QSize(880, 690)
+
+    def value(self) -> useq.WellPlatePlan | None:
+        """Return the current well plate plan, or None if the plan is uncalibrated."""
+        calib_plan = self.calibration_page.widget.value()
+        if not self._calibrated or not calib_plan:
+            return None
+
+        plate_plan = self.plate_page.widget.value()
+        if plate_plan.plate != calib_plan.plate:
+            warnings.warn("Plate Plan and Calibration Plan do not match.", stacklevel=2)
+            return None
+
+        return useq.WellPlatePlan(
+            plate=plate_plan.plate,
+            selected_wells=plate_plan.selected_wells,
+            rotation=calib_plan.rotation,
+            a1_center_xy=calib_plan.a1_center_xy,
+            well_points_plan=self.points_plan_page.widget.value(),
+        )
+
+    def setValue(self, value: useq.WellPlatePlan) -> None:
+        """Set the state of the wizard to a WellPlatePlan."""
+        self.plate_page.widget.setValue(value)
+        self.calibration_page.widget.setValue(value)
+        self.points_plan_page.widget.setValue(value.well_points_plan)
+
+    def save(self, path: str | None = None) -> None:
+        """Save the current well plate plan to disk."""
+        if not isinstance(path, str):
+            path, _ = QFileDialog.getSaveFileName(
+                self,
+                "Save Well Plate Plan",
+                "",
+                "JSON or YAML (*.json *.yaml *.yml)",
+            )
+        if path:
+            value = self.value()
+            if value is None:
+                return
+            if str(path).endswith((".yaml", ".yml")):
+                txt = value.yaml(exclude_unset=True)
+            else:
+                txt = value.model_dump_json(exclude_unset=True, indent=2)
+            Path(path).write_text(txt)
+
+    def load(self, path: str | None = None) -> None:
+        """Load a well plate plan from disk."""
+        if not isinstance(path, str):
+            path, _ = QFileDialog.getOpenFileName(
+                self,
+                "Load Well Plate Plan",
+                "",
+                "JSON or YAML (*.json *.yaml *.yml)",
+            )
+        if path:
+            self.setValue(WellPlatePlan.from_file(path))
```

Comment:
same here. I think yaml is only allowed if pyyaml is installed.  we either need to commit to depending on pyyaml here, leave this sort of detail to the "application" that incorporates this widget, or include the conditional fallback here if pyyamls isnt' installed